openstack/tempest/nova-tempest/templates/tempest-pod.yaml

```diff
@@ -11,7 +11,8 @@ spec:
   containers:
     - name: {{ .Chart.Name }}
       # As nova has no tempest plugin project we use standard tempest tests for compute -> use neutron-tempest-plugin image
-      image: {{ default "keppel.eu-de-1.cloud.sap/ccloud" .Values.global.registry}}/neutron-tempest-plugin:{{ default "latest" (index .Values (print .Chart.Name | replace "-" "_")).tempest.imageTag}}
+      # image: {{ default "keppel.eu-de-1.cloud.sap/ccloud" .Values.global.registry}}/neutron-tempest-plugin:{{ default "latest" (index .Values (print .Chart.Name | replace "-" "_")).tempest.imageTag}}
+      image: {{ default "keppel.eu-de-1.cloud.sap/ccloud" .Values.global.registry }}/{{ default .Chart.Name (index .Values (print .Chart.Name | replace "-" "_")).tempest.imageNameOverride }}-plugin-python3:{{ default "latest" (index .Values (print .Chart.Name | replace "-" "_")).tempest.imageTag}}
```

Comment:
We have to pass the python version for an image from Values because we need to run the same charts in Concourse